src/Upgrader/Console/EvaluateConsole.php

```diff
@@ -114,12 +114,10 @@ public function execute(InputInterface $input, OutputInterface $output): int
         );
 
         $codebaseSourceDto = $this->codebaseService->readCodeBase($codebaseRequestDto);
-
         $report = $this->codeComplianceService->analyze($codebaseSourceDto);
+        $this->reportService->save($report);
```

Comment:
I think if the report has no errors, that will be just an empty file. Have I understood correct, that you want with this saving you want to override the previous report with new one empty?

Reply:
Yes, you are right